doc/how_to/get_started.rst

```diff
@@ -337,109 +368,221 @@ can print the units found using:
 
-    template = we_TDC.get_template(unit_id0)
-    print(template.shape)
 
 
 .. parsed-literal::
 
-    WaveformExtractor: 32 channels - 10 units - 1 segments
-      before:96 after:128 n_per_units:500
-    (30, 224, 32)
-    (224, 32)
+    <spikeinterface.postprocessing.template_similarity.ComputeTemplateSimilarity at 0x7fb5013f6f40>
 
 
-``we_TDC`` is a have the ``WaveformExtractor`` object we can
-post-process, validate, and curate the results. With the
-``spikeinterface.postprocessing`` submodule, one can, for example,
-compute spike amplitudes, PCA projections, unit locations, and more.
 
-Let’s compute some postprocessing information that will be needed later
-for computing quality metrics, exporting, and visualization:
+Find out more about the available parameters and extensions
+`here <https://spikeinterface.readthedocs.io/en/latest/modules/postprocessing.html#available-postprocessing-extensions>`__.
 
-.. code:: ipython
+The calculations are saved in the ``extensions`` subfolder of the
+``SortingAnalyzer`` folder. Similar to the waveforms we can access them
+using ``get_extension`` and ``get_data``. For example, here we can make
+a historgram of spike amplitudes
+
+.. code:: ipython3
+
+    amplitudes = analyzer_TDC.get_extension("spike_amplitudes").get_data()
+    plt.hist(amplitudes, bins=50)
+    plt.show()
 
-    amplitudes = spost.compute_spike_amplitudes(we_TDC)
-    unit_locations = spost.compute_unit_locations(we_TDC)
-    spike_locations = spost.compute_spike_locations(we_TDC)
-    correlograms, bins = spost.compute_correlograms(we_TDC)
-    similarity = spost.compute_template_similarity(we_TDC)
 
 
-All of this postprocessing functions are saved in the waveforms folder
-as extensions:
+.. image:: get_started_files/get_started_52_0.png
 
-.. code:: ipython
 
-    print(we_TDC.get_available_extension_names())
+You can check which extensions have been saved (in your local folder)
+and which have been loaded (in your enviroment)…
+
+.. code:: ipython3
+
+    print(analyzer_TDC.get_saved_extension_names())
+    print(analyzer_TDC.get_loaded_extension_names())
 
 
 .. parsed-literal::
 
-    ['similarity', 'spike_amplitudes', 'correlograms', 'spike_locations', 'unit_locations']
+    ['random_spikes', 'waveforms', 'templates', 'noise_levels', 'template_similarity', 'spike_amplitudes', 'correlograms', 'spike_locations', 'unit_locations']
+    ['random_spikes', 'waveforms', 'noise_levels', 'templates', 'spike_amplitudes', 'unit_locations', 'spike_locations', 'correlograms', 'template_similarity']
+
+
+…or delete an extension…
+
+.. code:: ipython3
+
+    analyzer_TDC.delete_extension("spike_amplitudes")
 
+This deletes the extension’s data in the ``SortingAnalyzer`` folder.
 
-Importantly, waveform extractors (and all extensions) can be reloaded at
-later times:
+Importantly, ``SortingAnalyzers`` (and all extensions) can be reloaded
+at later times: (Here, spike_amplitudes is not loaded since we just
+deleted it)
```

Comment:
Maybe a mention that this needs to be run from the same folder or form the same placer or othrewise use an absolute path? 

What do you think?

Reply:
Maybe it's slightly clearer if we write the code as two lines?

Importantly, ``SortingAnalyzers`` (and all extensions) can be reloaded
at later times from their saved folders:

sorting_analyzer_path = 'analyzer_TDC_binary'
analyzer_loaded = si.load_sorting_analyzer(sorting_analyzer_path)

How about something like this?

Reply:
I'm wondering if the part of the question is where is `analyzer_TDC_binary`. We could emphasize that relativity by doing
```
sorting_analyzer_path = './analyzer_TDC_binary'
```

Although using the `./` is not completely beginner friendly. But for people that know it adds emphasis to location....

In our other docs that don't come from a `.py` conversion we often do 
```
path = 'where-i-want-my-data/analyzer_TDC_binary'
```
or some other descriptive path, but that doesn't really work here.

Reply:
Both adding a second line and using the linux notation for current directoy seems like an improvement to me.

Reply:
It works for Windows too with `Path('r.\abc\')` :) or even `Path('./abc/'), but I think you're right in that I could easily google what does the period at the beginning of a file path mean and find out. So adding this is clearer and even for a beginner it should be relatively easy to google to find out.

Reply:
Perfect, I've added both